### ISSUE TITLE
Modelling issue: package format

### ISSUE THREAD

**Issue by timgdavies** [openownership/data-standard#15]
> An issues related to the [draft conceptual model](https://github.com/openownership/data-standard/issues/7)

We will need to decided upon a packaging format for bulk releases of data. 

This will involve identifying the meta-data that should be provided, and how statements should be packaged together.

We will look at the [Data Package Specification](http://specs.frictionlessdata.io/data-packages/) as a reference point for this. 

We note that  the GLEIF has ‘Delta’ field to indicate when data is since for bulk files.

**Comment by jpmckinney** [comment]
Can you link to the docs for that `Delta` field?

**Comment by ScatteredInk** [comment]
Google have just added guidelines for marking up [public datasets](https://developers.google.com/search/docs/data-types/datasets) - this could be useful in terms of discoverability etc and the guidelines reuse existing vocabularies. 

`citation` is one that jumps out as helpful for the original research use-cases, where maintaining attribution will be important.

**Comment by jpmckinney** [comment]
That link reuses Schema.org's [Dataset](http://schema.org/Dataset) and related classes, which are based on [DCAT](http://www.w3.org/TR/vocab-dcat/).

**Comment by ScatteredInk** [comment]
@jpmckinney Yes: in the accompanying [blog](https://research.googleblog.com/2017/01/facilitating-discovery-of-public.html), they also suggest contributing to Schema.org, so one option would be to suggest any missing vocabulary from Data Package gets added there.

**Comment by timgdavies** [comment]
This is not covered in the beta release - but should be addressed by any release candidate.

**Comment by kd-ods** [comment]
For the record, this is a query and my response to a prospective publisher (around bulk downloads and API options in BODS v0.2):

" **Regarding the publication method:
a. It appears, the whole published data is one huge file, which is getting larger and larger. Wouldn’t it be possible to publish just the list of all statement identifiers, or better statement URI-s in order to keep the size of the file smaller? Each single statement would be permanently accessible via its’ URI.**

Effectively, this sounds like you'd be developing an API via which a valid BODS dataset could be constructed. Compared with downloading a single (admittedly large) file containing the complete dataset, the API pushes extra work onto the data user. But it's a perfectly reasonable approach."
